intro-HPC/ch_introduction.tex

```diff
@@ -272,8 +315,20 @@ \subsection{What programming languages can I use?}
 \operatingsystem. Supported and common programming languages on the \hpc are
 C/C++, FORTRAN, Java, Perl, Python, MATLAB, R, etc.
 
-Supported and commonly used compilers are GCC, clang, J2EE and Intel Cluster
-Studio.
+Supported and commonly used compilers are 
+%TODO insert compilers for other sites
+\ifantwerpen
+GCC, clang, J2EE and Intel Cluster Studio.
+\fi
+\ifleuven
+GCC, Intel and PGI. 
+\fi
+\ifbrussel
+GCC, clang, J2EE and Intel Cluster Studio.
+\fi
+\ifgent
+GCC, clang, J2EE and Intel Cluster Studio.
```

Comment:
The Leuven line should contain `GCC, Intel and PGI` afaik, since you know what you have. But in Ghent, you should probably only mention `GCC and Intel`, which is what @boegel was referring to.